### DISCUSSION TITLE
Dask clusters on new ADE

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#819] [Platform]
Hey folks, I see that on the new ADE there is a dask dashboard tab. Is there any documentation on how to use that? I am used to using dask in my notebooks and making a client using:

```
from dask.distributed import Client, progress
client = Client(threads_per_worker=4, n_workers=1)
client
```

and I thought that maybe I could use the tab to access the local cluster urls, but it doesn't seem to work. I would love to know how to use it.

**Comment by mccabete** [comment]
When I inject  a cluster into an exisitng notebook, like so:
```
# from dask.distributed import Client

# client = Client("tcp://127.0.0.1:36779")
# client
```
 it seems to just give me cancellation errors.

**Reply by wildintellect** [reply]
@mccabete I looked into this. Are you using the Pangeo workspace? Dask tools are installed as part of Pangeo but there is not specific backend support for this in MAAP. I'll bring this up with the Platform team and we might hide this until we have time to figure out how to make it work for at least a local cluster.

**Reply by mccabete** [reply]
I was using the pangeo workspace, but I probably had a custom env on top of that. Didn't run into anything that made me think it was a dependency issue though. The sense I got from the slack conversation is that as you mention above the Dask tab isn't ready for use yet.
